### PR TITLE
Made a bunch of edits!

### PR DESCRIPTION
**Pull request by rjpecoraro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,8 +44,15 @@ While many proteins fold into regular conformations which can be easily analyzed
...
 
-------------------------
+ We welcome the members of open-source community to extend the functionalities of TBD, submit feature requests and report bugs.
+ 
+ ### Feature Request:
+ If you would like to suggest a feature or start a discussion on possible extension of TBD, please feel free to <a href="https://github.com/Intrinsically-Disordered/TBD/issues/new">raise an issue</a>.
+ 
+ ### Bug Report:
+ If you would like to report a bug, please follow <a href="https://github.com/Intrinsically-Disordered/TBD/issues/new">this link</a>.
+
```
Comment (jiaweih) [inline README.md:54]: So the bug report and feature request are supposed to be the same link?

File: CONTRIBUTIONS.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+git push
+```
+
+### Step 4. Generating pull request
+
+The final step is to generate a pull request. The instructions for opening a pull request are outlined in this <a href="https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request">tutorial</a>
+
+
+## Things to note
+To achieve the purpose of continous integration, <code>hardy</code> utilizes <code>travis CI</code>. travis CI is configured to perform PEP8 compliance test (flake8) along with running unittests. Any failure in compliance would result in build error or build failure on travis CI.
```
Comment (jiaweih) [inline CONTRIBUTIONS.md]: Need to update `hardy`

File: README.md (hunk 1 above)
Comment (rjpecoraro) [reply README.md:54]: That's how HARDy did it, but I feel like that makes sense

File: CONTRIBUTIONS.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```
+conda install flake8
+cd TBD
+flake8
+```
+
+For unittest,
+```
+cd TBD
+python -m unittest discover
```
Comment (jiaweih) [inline CONTRIBUTIONS.md:48]: We are not using unittest in this project

Comment (rjpecoraro) [reply CONTRIBUTIONS.md:48]: OK cool, wasn't sure what tests we're using since nothing was in the repository. What tests are you using instead?

Comment (rjpecoraro) [reply CONTRIBUTIONS.md:48]: Actually, I'm not convinced that the main interface to run the model won't use it. I'm happy to write those tests, but I think I need to see how the model works first

File: CONTRIBUTIONS.md (hunk 2 above)
Comment (rjpecoraro) [reply CONTRIBUTIONS.md]: Done!

File: CONTRIBUTIONS.md (hunk 3 above)
Comment (jiaweih) [reply CONTRIBUTIONS.md:48]: I uploaded the code two days ago. The tests were written too.

**Review by jiaweih — APPROVED** [review]

**Pull request merged**